### DISCUSSION TITLE
Ordered List Expansion

### DISCUSSION THREAD

**Discussion by laowangdefangjian** [ueberdosis/tiptap#5330] [General Questions] [answered]
I hope that components such as sequence tables, unordered lists, and details can be extended to form each row as a block, so that they can all be dragged and dropped with a controller
Please provide me with a simple example in the extension. Thank you for your response

**Comment by nperez0111** [comment, marked as answer]
Our example of this is the notion like template. Go to https://templates.tiptap.dev

**Reply by laowangdefangjian** [reply]
But what should I do if the example you gave me shows that the entire ul is a block with only one drag handle, and each li can be dragged?

**Reply by nperez0111** [reply]
you can see in the code how the drag handle is attached to a block if you look at the template's source code after signing into a free tiptap account and connecting your github account.
